code-challenges.js

```diff
@@ -14,29 +14,125 @@ const fruit2 = "banana"
 const fruit3 = "cherry"
 const fruit4 = "kiwi"
 
+//create a function (const <name> = () => {}) -> bigString
+//inputs two strings (string1 , string2) --> output (return) bigger string
+// I think of comparing -> decision making-conditionals (boolean) - if(this){do this}
+//what if strings are = ?
+//invoke the function using console.log() - input the test variables, and other data.
+
+//could also use string interpolation to say how many characters each string was, or how much bigger the string was from the other
+//`${string1} was bigger by ${string1.length-string2.length}`
+
+const bigString = (string1 , string2) => {
+  if(string1 > string2){
+    return string1
+  }else if(string1 === string2){
+    return "They are the same!"
+  }else{
+    return string2
+  }
+}
+console.log(bigString(fruit2 , fruit3))
+console.log(bigString(fruit1 , fruit4))
+console.log(bigString("fruit2" , "fruit3"))
+console.log(bigString(1234 , "abcdefghi"))
+
+//My outputs:
+// cherry
+// kiwi
+// fruit3
+// abcdefghi
+
 
 // --------------------1) Create a function that takes in any given number and determines if the number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
 const temp1 = 42
 const temp2 = 350
 const temp3 = 212
 
-
+//make a function atTemp
+//it has to make a decision based on input numbers
+//conditionals-> if(this){then do this} for each tempRange + ANY given number
+//the output will return the number given and a string -> string interpolation `${}`
+//invoke the function using console.log()
+
+const atTemp = (temp) => {
+  if(temp > 212){
+    return `${temp} is above boiling point.`
+  }else if(temp === 212){
+    return `${temp} is at boiling point.`
+  }else{
+    return `${temp} is below boiling point.`
+  }
+}
+console.log(atTemp(temp1));
+console.log(atTemp(temp2));
+console.log(atTemp(temp3));
+console.log(atTemp(8990));
+console.log(atTemp(0));
+// my outputs:
+// 42 is below boiling point.
+// 350 is above boiling point.
+// 212 is at boiling point.
+// 8990 is above boiling point.
+// 0 is below boiling point.
+//I wanted to try negative numbers but wasn't sure how to do that since it recognized the "-" symbol as minus (math)
```

Comment:
This is excellent! Thanks for taking me through your thought process. Great use of this space to come up with the terms you'll need to be looking up.

Reply:
Interesting point in the negative numbers maybe declare the negative number as a variable and call on the variable?